lib/setupmanagers/qemuhck/network_manager.rb

```diff
@@ -156,15 +168,28 @@ def transfer_device_command(device_name, transfer_net, share_path, qemu_replacem
         net_base = "#{transfer_net}.0/24"
         smb_server = "#{transfer_net}.4"
 
-        netdev_options = "net=#{net_base},smb=#{path},smbserver=#{smb_server},restrict=on,ifname=@net_if_name@"
+        net_smb_private = replace_string_recursive('@workspace@/smb_@run_id@', qemu_replacement_list)
```

Comment:
In my setup, I have a long workspace path and during the testing of this PR smbd failed with the error `messaging_dgm_ref failed: File name too long`. smbd creates a socket in this directory and Linux limits socket path to 108 chars long, so need to make sure with this

Reply:
Isn't it better to relocate the workspace? We already have other socket files in workspaces.

Reply:
> > We already have other socket files in workspaces.
> 
> which one? viofs socket moved to /tmp we have the same issue.

I missed it's in `/tmp`. I'll relocate smb sockets to `/tmp` too.